src/components/button/button.scss

```diff
@@ -31,6 +31,28 @@
   @include md-fab($md-mini-fab-size, $md-mini-fab-padding)
 }
 
+[md-icon-button] {
+  @extend %md-button-base;
+
+  min-width: 0;
+  padding: 0;
+
+  width: 40px;
```

Comment:
Would you mind putting this in a Sass variable?  All other button sizes are variables in _button_base.scss, so dense layouts can adjust as necessary.